### PR TITLE
Refactor components properties

### PR DESCRIPTION
**Pull request by MGREMY**

### CODE REVIEW

**Comment by SutuSebastian** [comment]
One quick thing, I just checked the preview and the sidebar doesn't go all the way to the bottom of the screen:

Screenshot 2024-04-08 at 17 35 50

**Comment by SutuSebastian** [comment]
But so nice u fixed that animation flicker on reload 🔥

**Comment by MGREMY** [comment]
> One quick thing, I just checked the preview and the sidebar doesn't go all the way to the bottom of the screen:
> 
> Screenshot 2024-04-08 at 17 35 50

It's fixed now !

File: apps/docs/src/app/pages/button-page/button-page.component.html [hunk 1]
```diff
@@ -1,226 +1,229 @@
...
+      </flowbite-button>
+    </div>
+  </flowbite-demo-example>
+  <flowbite-demo-example title="Button with label" [codes]="[examples[8]]">
+    <div class="flex flex-wrap gap-2">
+      <flowbite-button>
+        Messages
+        <flowbite-badge
+          [customStyle]="badgeCustomTheme"
+          [isIconOnly]="'enabled'"
```
Comment (MGREMY) [inline apps/docs/src/app/pages/button-page/button-page.component.html:153]: `isIconOnly="true"`

**Pull request merged**